.travis.yml

```diff
@@ -1,6 +1,17 @@
+sudo: false
+os:
+  - linux
```

Comment:
I think yes, but we need same line in `matrix` AFAIR, so I prefer hold `os` in main section too.